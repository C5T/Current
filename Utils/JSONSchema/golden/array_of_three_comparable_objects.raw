{"Array":{"element":{"Object":{"field_schema":[["x",{"String":{"values":{"b":1,"a":2},"counters":[[1,"b"],[2,"a"]],"instances":3,"nulls":0}}]],"field_index":{"x":0},"instances":3,"nulls":0}},"instances":1,"nulls":0}}