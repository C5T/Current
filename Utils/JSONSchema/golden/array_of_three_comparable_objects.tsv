Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	Object	3	0	1 field	x
Schema[].x	String	3	0	2 distinct values, "a":2, "b":1
